Player/Player.gd

```diff
@@ -0,0 +1,26 @@
+extends CharacterBody2D
+
+
+const speed = 200
+@onready var anim = get_node("AnimatedSprite2D")
+
+func _ready():
+	anim.play("Idle")
+
+func _process(delta):
+	var velocity = Vector2()
+	if Input.is_action_pressed("move_right"):
+		velocity.x += 1
+	if Input.is_action_pressed("move_left"):
+		velocity.x -= 1
+	if Input.is_action_pressed("move_down"):
+		velocity.y += 1
+	if Input.is_action_pressed("move_up"):
+		velocity.y -= 1
+	if velocity.length() > 0:
+		velocity = velocity.normalized() * speed
+		anim.play("Running")
+	else:
+		anim.play("Idle")
+	position += velocity * delta
```

Comment:
Humm, since you are using a `CharacterBody2D` you should use the `move_and_slide` method

Reply:
But if you want to keep your code, you should put this code in the physics function:

https://docs.godotengine.org/en/stable/tutorials/scripting/idle_and_physics_processing.html